### ISSUE TITLE
Allow saving same new key to multiple NFC tokens

### ISSUE THREAD

**Issue by hillbrad** [open-keychain/open-keychain#1966]
It is a bit scary to save a key that one expects to use to just one NFC Token, but if you have multiple tokens there seems to be no way to save the same newly generated key to multiple devices?  With  a Yubikey NEO it seems one can do the backup and then manually import the key into the Yubikey using command line tools and the USB interface, but how would one provision the same key to multiple Fidesmo cards?  It would be nice to have the option after saving the key one device to save it immediately to another, or to copy a key from an existing device to a backup device.

**Comment by Valodim** [comment]
Yup, sounds reasonable. Not sure if we'll get to it anytime soon though :)
